packages/system/coredns/values.yaml

```diff
@@ -0,0 +1,3 @@
+coredns:
+  service:
+    clusterIP: "10.95.0.10"
```

Comment:
Let's move these values definition to `coredns` helmchart resources in `kubernetes` application, we might want to reuse this helm chart later for management cluster as well.